romancal/resample/resample_utils.py

```diff
@@ -102,3 +102,31 @@ def make_output_wcs(
     )
 
     return wcs, pscale * 3600.0, pscale_ratio
+
+
+def compute_var_sky(model) -> None:
+    """
+    Add sky variance array to a datamodel.
+
+    Parameters
+    ----------
+    model : `ImageModel`
+        A datamodel to which the sky variance array will be added.
+
+    Returns
+    -------
+    None
+    """
+
+    # convert NaNs to zeros
+    data = np.nan_to_num(model["data"], nan=0.0)
+    median_data = np.median(data)
```

Comment:
```suggestion
    median_data = np.nanmedian(model["data"])
```
Wouldn't `nanmedian` be more accurate?

Reply:
Resolving in favor of https://github.com/spacetelescope/romancal/pull/1691#discussion_r2322534242